media/linux/pds-sqlite3-queries/create-training-rosters.py

```diff
@@ -75,134 +109,53 @@ def _dt_to_int(datetime):
             continue
 
         for req in m[key]:
-            if(req['description'] != training_to_find):
+            if(req['description'] != training_to_find['pds_type']):
                 continue
             reqcount += 1
             mem = pretty_member(m)
             sd = req['start_date']
-            ed = req['end_date']
             mid = mem['mid']
-            if sd not in out:
-                out[sd] = dict()
-            if ed not in out[sd]:
-                out[sd][ed] = dict()
-            if mid not in out[sd][ed]:
-                out[sd][ed][mid] = list()
-            out[sd][ed][mid].append({
+            if mid not in out:
+                out[mid] = dict()
+            if sd not in out[mid]:
+                out[mid][sd] = list()
+            out[mid][sd].append({
                 'mid'           :   mid,
                 'name'          :   mem['name'],
-                'phone'         :   mem['phones'][0],
                 'email'         :   mem['email'],
-                'description'   :   req['description'],
+                'phone'         :   mem['phones'][0],
                 'start_date'    :   sd,
-                'end_date'      :   ed,
-                'result'        :   req['result'],
+                'end_date'      :   req['end_date'],
+                'stage'         :   req['result'],
+                'active'        :   mem['active'],
+                'weekend'       :   mem['weekend'],
+                'weekday'       :   mem['weekday'],
+                'homebound'     :   mem['homebound'],
                 'note'          :   req['note'],
             })
-            #print(m['first']+' '+m['last']+": "+str(recent['end_date']))
+            print(m['first']+' '+m['last']+": "+str(sd))
     
-    print(f"Found {reqcount} training records")
-    return out
+    if out is None or len(out) == 0:
+        print("No trainings of type: {train} found".format(train=training_to_find['title']))
```

Comment:
Use `f` strings, not `.format()`.

Also, use logging functionality -- not `print`.